### ISSUE TITLE
igraph.to.gexf error with edge attributes

### ISSUE THREAD

**Issue by balachia** [gvegayon/rgexf#38]
I've run into a reversion caused by changes in 39d24409c7c641a1e62bdf518ff8ca46ea6172b5. When a graph (igraph) contains non-standard edge attributes, it cannot be converted to gexf. E.g. try running the example below:

```
df <- data.frame(from=c(1, 1, 2),
                 to=c(2, 3, 3),
                 flavor=c('yum', 'yum', 'yuck'))

g <- igraph::graph_from_data_frame(df)
gexf <- rgexf::igraph.to.gexf(g)
```

The apparent offending lines are below, which appear to drop all edge attributes from `edges` and then attempt to subset named attributes out of `edges`, which only fails if those attributes exist in the original graph.

https://github.com/gvegayon/rgexf/blob/dd222bf23b87fee00d6c5c6f918812e0338e5005/R/igraph.R#L58-L60
https://github.com/gvegayon/rgexf/blob/dd222bf23b87fee00d6c5c6f918812e0338e5005/R/igraph.R#L69-L72

From skimming the code, this seems to have been caused by a change from relying on a `tmpedges` copy of `edges` for the subsetting to using `edges` itself. I haven't spent enough time on the code here to figure out why dropping the `tmpedges` copy was necessary or what a fix might be.

**Comment by gvegayon** [comment]
I'll check it out. Thanks for the report

**Comment by gvegayon** [comment]
Fixed in #43
